packages/framework/src/Model/Feed/DailyFeedCronModule.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Feed;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Setting\Setting;
+use Shopsys\Plugin\Cron\IteratedCronModuleInterface;
+use Symfony\Bridge\Monolog\Logger;
+
+class DailyFeedCronModule implements IteratedCronModuleInterface
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Feed\FeedFacade
+     */
+    private $feedFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    private $domain;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Setting\Setting
+     */
+    private $setting;
+
+    /**
+     * @var \Symfony\Bridge\Monolog\Logger
+     */
+    private $logger;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Feed\FeedExportDataQueue|null
```

Comment:
Can null even happen? you set it to FeedExportDataQueue in constructor

Reply:
It cannot be null, thanks. Probably a leftover from previous refactoring :)